x-pack/plugins/rollup/server/routes/api/index_patterns.js

```diff
@@ -25,26 +26,36 @@ export function registerFieldsForWildcardRoute(server) {
       pre: [ licensePreRouting ],
       validate: {
         query: Joi.object().keys({
-          pattern: Joi.string(),
+          pattern: Joi.string().required(),
           meta_fields: Joi.array().items(Joi.string()).default([]),
-          fields: Joi.array(),
           params: Joi.object().keys({
             rollup_index: Joi.string().required(),
-          }).default({})
+          }).required()
         }).default()
       }
     },
     handler: async (request) => {
       const {
+        pattern,
         meta_fields: metaFields,
-        fields,
         params,
       } = request.query;
 
-      const rollupIndex = params.rollup_index;
-      const callWithRequest = callWithRequestFactory(server, request);
+      // Format call to standard index pattern `fields for wildcard`
+      const standardRequest = {
+        url: `/api/index_patterns/_fields_for_wildcard?${querystring.stringify({ pattern, meta_fields: metaFields })}`,
```

Comment:
`server.inject` does not support passing query params as part of the request object, hence the stringify here